packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,19 +7,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [50.0.0]
+
 ### Added
 
 - Add additional metadata for batch metrics ([#5488](https://github.com/MetaMask/core/pull/5488))
   - Add `delegationAddress` to `TransactionMetadata`.
   - Add `NestedTransactionMetadata` type containing `BatchTransactionParams` and `type`.
   - Add optional `type` to `TransactionBatchSingleRequest`.
+- Verify EIP-7702 contract address using signatures ([#5472](https://github.com/MetaMask/core/pull/5472))
+  - Add optional `publicKeyEIP7702` property to constructor.
+  - Add dependency on `^5.7.0` of `@ethersproject/wallet`.
 
 ### Changed
 
+- Bump `@metamask/accounts-controller` peer dependency to `^26.1.0` ([#5481](https://github.com/MetaMask/core/pull/5481))
```

Comment:
This is a breaking change.

But, do we need it? Typically we don't bump peer dependencies unless we've added a new messenger action/event (or we are bumping by a major).

Reply:
Are these bumps not automatic whenever we release a new version of a controller, including minor?

Reply:
Also, I understood only major peer dependencies are breaking, since a consumer who has `^26.0.0` will automatically get `26.1.0` when they add the new transaction controller version?

Reply:
Hmm, I don't think the consumer would automatically get 26.1.0. Yarn/NPM doesn't consult peer dependencies to know what to install automatically; that's up to the consumer. So I believe that the consumer would receive a peer dependency warning, and then they would need to bump their dependency on `accounts-controller` to resolve that.

Reply:
Done.

Reply:
Oh I think what I meant also was that we can merge those two entries, since the consumer will only see an update to 26.1.0 and this effectively overrides the original change which introduced the 26.0.0 peer dependency. I'll suggest another change.